src/dotnet/APIView/APIViewWeb/Repositories/DevopsArtifactRepository.cs

```diff
@@ -37,6 +37,11 @@ public DevopsArtifactRepository(IConfiguration configuration, IHttpContextAccess
             _devopsAccessToken = Convert.ToBase64String(System.Text.Encoding.ASCII.GetBytes(string.Format("{0}:{1}", "", _configuration["Azure-Devops-PAT"])));
             _hostUrl = _configuration["APIVIew-Host-Url"];
             _pipelineNameCache = cache;
+
```

Comment:
Since this code is configuring a `static readonly` variable, I think it would be best to put it in a static ctor, like so:

```c#
static readonly HttpClient _devopsClient;

static DevopsArtifactRepository() {
    var client = new HttpClient();
    client.DefaultRequestHeaders.Accept.Clear();
    client.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
    client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", _devopsAccessToken);
    _devopsClient = client;
}
```

Reply:
I not sure how that will work since the class DevopsArtifactRepository is injected via dependency injection and the other variables in the class are not static.

Reply:
> I not sure how that will work since the class DevopsArtifactRepository is injected via dependency injection and the other variables in the class are not static.

Sorry you are correct, I didn't notice that `_devopsAccessToken` relies on `_configuration` which is a ctor parameter.  In this case, I believe the only correct fix to change `_devopsClient` from a static field to an instance field.